content/pt-br/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -0,0 +1,291 @@
+| TCP      | Entrada   | 30000-32767 | Serviços NodePort†    | Todos                     |
+
+† Intervalo padrão de portas para o [Serviços NodePort](/docs/concepts/services-networking/service/).
+
+Quaisquer portas marcadas com * podem ser substituídas. Dito isso, tenha certeza de que qualquer porta que você tenha customizado esteja aberta.
+
+Embora portas etcd sejam inclusas nos nós da camada de gerenciamento, você também pode hospedar seu próprio cluster etcd externamente, ou em portas customizadas. 
+
+O plugin de rede dos pods que você utiliza também pode requer que algumas portas estejam abertas. Dito que essas portas podem diferir dependendo do plugin, por favor leia a documentação dos plugins sobre quais portas serão necessárias abrir.
+
+## Instalando agente de execução {#instalando-agente-de-execucao}
+
+Para executar os contêineres nos Pods, o Kubernetes utiliza um 
+{{< glossary_tooltip term_id="container-runtime" text="agente de execução" >}}.
+
+{{< tabs name="container_runtime" >}}
+{{% tab name="Nós Linux" %}}
+
+Por padrão, o Kubernetes utiliza a {{< glossary_tooltip term_id="cri" text="interface do agente de execução">}} (CRI) para interagir com o seu agente de execução de contêiner escolhido.
+
+Se você não especificar nenhum agente de execução, o kubeadm irá tentar identifica-lo automaticamente através de uma lista dos sockets Unix mais utilizados. A tabela a seguir lista os agentes de execução e os caminhos dos sockets a eles associados.
+
+{{< table caption = "Agentes de execução e seus caminhos de socket" >}}
+| Agente de execução    | Caminho do socket Unix        |
+|------------|-----------------------------------|
+| Docker     | `/var/run/dockershim.sock`        |
+| containerd | `/run/containerd/containerd.sock` |
+| CRI-O      | `/var/run/crio/crio.sock`         |
+{{< /table >}}
+
+<br />
+Se tanto o Docker quanto o containerd forem detectados no sistema, o Docker terá precedência. Isso acontece porque o Docker, desde a versão 18.09, já incluí o containerd e ambos são detectaveis mesmo que você só tenha instalado o Docker. Se outros dois ou mais agentes de execução forem detectados, o kubeadm é encerrado com um erro.
+
+O kubelet se integra com o Docker através da implementação CRI `dockershim` já inclusa.
+
+Veja a página dos [agentes de execução](/docs/setup/production-environment/container-runtimes/)
+para mais detalhes.
+{{% /tab %}}
+{{% tab name="Outros sistemas operacionais" %}}
+Por padrão, o kubeadm utiliza o {{< glossary_tooltip term_id="docker" >}} como agente de execução.
+O kubelet se integra com o Docker através da implementação CRI `dockershim` já inclusa.
+
+Veja a página dos [agentes de execução](/docs/setup/production-environment/container-runtimes/)
+para mais detalhes.
+{{% /tab %}}
+{{< /tabs >}}
+
+
+## Instalando o kubeadm, kubelet e o kubectl
+
+Você instalará esses pacotes em todas as suas máquinas:
+
+* `kubeadm`: o comando para iniciar o cluster.
```

Comment:
iniciar here seems like startup. As in English is "bootstrap", probably "o comando para criar o cluster" seems better

Reply:
Indeed. "Criar" seems more appropriate.